include/platform/mir/input/event_builder.h

```diff
@@ -38,8 +38,14 @@ class EventBuilder
 public:
     EventBuilder() = default;
     virtual ~EventBuilder() = default;
+    /// Timestamps in returned events are automatically calibrated to the Mir clock
     using Timestamp = std::chrono::nanoseconds;
 
+    /// True resets timestamp calibration to it's initial state, so the next event with a timestamp will be at the
+    /// "current" time (according to the Mir clock) and subsequent events will be relative to that. False disables
+    /// calibration, so future events will not have their timestamps modified.
+    virtual void calibrate_timestamps(bool enable) = 0;
```

Comment:
Not sure about this name (too close to `calibrate_timestamp()`), nor the `bool` parameter (which is always `false`).

Reply:
Better?